MIP113/MIP113.md

```diff
@@ -293,8 +293,6 @@ List of current ADs:
 | VoteWizard | [EA Address (etherscan)](https://etherscan.io/address/0x9E72629dF4fcaA2c2F5813FbbDc55064345431b1)<br>[Signature](https://etherscan.io/verifySig/19275) | [Contract 1 (etherscan)](https://etherscan.io/address/0x867761533306C5b8476507a982d51D04fBf9d4Fd)<br>Followed AVC: **KISS AVC**<br>[Signature](https://etherscan.io/verifySig/19276) | [Contract 2 (etherscan)](https://etherscan.io/address/0x966Bf9Dc96c2f8d4F67bC23C8A6bFfB722046238)<br>Followed AVC: **Growth AVC**<br>[Signature](https://etherscan.io/verifySig/19277) | [Link](https://forum.makerdao.com/t/votewizard-ad-recognition-submission/20973) |
 | Navigator | [EA Address (etherscan)](https://etherscan.io/address/0x11406a9CC2e37425F15f920F494A51133ac93072)<br>[Signature](https://etherscan.io/verifySig/19371) | [Contract 1 (etherscan)](https://etherscan.io/address/0x636Ff7CA03D91D8C5614394b4cC78C642a5e2Ce5)<br>Followed AVC: **ReFi AVC**<br>[Signature](https://etherscan.io/verifySig/19369) | [Contract 2 (etherscan)](https://etherscan.io/address/0x6235c806b6A7752e47A9adc815Ba343DF62A4A0a)<br>Followed AVC: **Growth AVC**<br>[Signature](https://etherscan.io/verifySig/19370) | [Link](https://forum.makerdao.com/t/navigator-ad-recognition-submission/20993) |
 | Harmony | [EA Address (etherscan)](https://etherscan.io/address/0xF4704Aa4Ad22cAA2A3Dd7A7C529B4C32f7A421F2)<br>[Signature](https://etherscan.io/verifySig/19344) | [Contract 1 (etherscan)](https://etherscan.io/address/0xaaDc617b289f828CC218bE5A47A1AD19b9c857Ac)<br>Followed AVC: **Growth AVC**<br>[Signature](https://etherscan.io/verifySig/19346) | [Contract 2 (etherscan)](https://etherscan.io/address/0xc02438A40fB1EbE254CEb21C6C2009a69C264Cf7)<br>Followed AVC: **Resiliency AVC**<br>[Signature](https://etherscan.io/verifySig/19345) | [Link](https://forum.makerdao.com/t/harmony-ad-recognition-submission/20987) |
-| Bulwark | [EA Address (etherscan)](https://etherscan.io/address/0xeb793A9A442997C43119eBedfD0F2f757f23334c)<br>[Signature](https://etherscan.io/tx/0x7b62eeb34f4719bfa409ee2fffa67a400ea867e3bfa03cde6a103af9256cb867) | [Contract 1 (etherscan)](https://etherscan.io/address/0x516473ce5286d5D9eA3485e3F08793e7595a8bD6)<br>Followed AVC: **KISS AVC**<br>[Signature](https://etherscan.io/tx/0x7c404d7afd40c8d766205ab7411214092771dfa361bd85c3e7088c02b0e8ae66) | [Contract 2 (etherscan)](https://etherscan.io/address/0xaA6834D882f5bb983f226705b350DAA52Cf2AA30)<br>Followed AVC: **Risk AVC**<br>[Signature](https://etherscan.io/tx/0x12d5b8516e39b13a404835c84bd136576db2d5983165352dc5da3403b0e5509d) | [Link](https://forum.makerdao.com/t/cd-recognition-submission-bulwark/21013) |
```

Comment:
fix this deletion, Bulwark added after Rune made his forum post - this is an active element so we can fix it